HUD/SetMinimapComponent.md

```diff
@@ -5,17 +5,28 @@ ns: HUD
 
 ```c
 // 0x75A9A10948D1DEA6 0x419DCDC4
-Any SET_MINIMAP_COMPONENT(int p0, BOOL p1, int p2);
+void SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int p2);
```

Comment:
Changing return values will have arbitrary ABI consequences. Are you sure no compiled C# code is using this?

If not (which, fyi, it's impossible not to be), _please retain the original type signature_.